### DISCUSSION TITLE
Max Steiner (BAUHAUS) [UR]

### DISCUSSION THREAD

**Discussion by Michalx11** [secretcowlevel/DoomtrooperPublic#42] [Card Balance]
Nobody plays poor Max. He needs some spruce up for sure. How about:

Max Steiner 
4/7/9/8
1 Action - target Venusjan Ranger gains BOUNTY +1 until beginning of your next turn 
(SImilar to Gallagher but BOUNTY instead of SLAYER)

**Comment by TomekPL-1** [comment]
Etoiles Mortant (BAUHAUS) [C]
ETOILESMORTANT
Enter your SQUAD with one counter. Once during your turn you can remove one counter to gain until end of turn one option:
-POWERIMMUNITY
-BLESSED
-Gains +3 to [F], [S] and [A] when combating DARKLEGION Warriors and HERETICs.
F: 4 S: 2 A: 4 V: 4

Venusian Marshal (BAUHAUS) [U]
When he enter your Squad you can add one counter on every warrior. Once per turn as one ACTION he can perform one from below action:
-Your other non-UNIQUE Warriors gain +2 to [F] , [S] and [A] until beginning of your nex turn.
-Put one counter on target Warrior
F: 6 S: 5 A: 6 V: 7

VENUSIAN RANGER Bauhaus [U]
VENUSIANRANGER
Enter your SQUAD with one counter.
Once during your turn you can remove one counter to chose one option:
-During next combat gain +2 to their [F] , [S] , and [A] (this is cumulative)
-During next combat gain FIRST STRIKE
F: 4 S: 3 A: 4 V: 4

Max Steiner Bauhaus [UR]
UNIQUE
When your Venusian Rangers become WOUNDED put one counter on him and he have +5 to [F] and [S] until healed
F: 4 S: 7 A: 9 V: 8
